moped-editor/src/views/staff/EditStaffView.js

```diff
@@ -59,13 +59,33 @@ const EditStaffView = () => {
    * @param {Object} data - The data returned from user form to submit to the Moped API
    */
   const onFormSubmit = (data) => {
+    const {
+      first_name,
+      last_name,
+      title,
+      email,
+      password,
+      roles,
+      workgroup_id,
+    } = data;
+
+    const payload = {
+      first_name,
+      last_name,
+      title,
+      email,
+      password,
+      roles,
+      workgroup_id,
+    };
+
```

Comment:
is this restructuring doing something important or were you just debugging here?

Reply:
The form submission broke since I was lazily passing the whole user payload through the form which included the `moped_workgroup` data that the backend no longer needs. I'll take a closer look at this loose end when I check the workgroup error mentioned below. I think that we can trim down the data requested to populate this form. 🙏

Reply:
ah, of course, you're being explicit about what values to handle—i just misinterpreted. this looks fine to me!

Reply:
Cool! I kept it around for future eyes. 👀